app-backup/ccollect/ccollect-0.8-r1.ebuild

```diff
@@ -28,16 +30,19 @@ src_compile() {
 }
 
 src_install() {
+	hprefixify ccollect.sh
```

Comment:
This needs a revbump, right? Or does it fail to install thanks to QA checks on prefix?

Reply:
It does not fail but it call the shell outside prefix.

Reply:
New revision then.